controllers/articles.js

```diff
@@ -33,8 +32,28 @@ function sendArticles(req, res, next) {
       });
   }
 
+  function getCommentById(req, res, next) {
+
+    const  {article_id }= req.params;
+
+
+    fetchCommentsById(article_id)
+      .then((article) => {
+        
+
+        res.status(200).send({
+          article,
```

Comment:
Is article the best key to send the comments back on here or would comments be more descriptive?